### ISSUE TITLE
Add timestamps to btcpayserver and nbxplorer log entries

### ISSUE THREAD

**Issue by IainDouglas** [btcpayserver/btcpayserver-docker#147]
I recently installed BTCPayserver using docker. While trying to investigate an issue I noticed that the generated_btcpayserver_1 and  generated_nbxplorer_1 container log files did not have timestamps for log entries. 

Please add timestamps to the log entries for btcpayserver and nbxplorer.

**Comment by NicolasDorier** [comment]
have you tried `docker logs --timestamps` ?

**Comment by IainDouglas** [comment]
Yes, that definitely works but gets irritating real quick having to remember which containers need it and which don't.
